src/libcore/tests/slice.rs

```diff
@@ -1127,12 +1127,13 @@ fn sort_at_index() {
                 // Sort in ascending order.
                 for pivot in 0..len {
                     let mut v = orig.clone();
-                    v.sort_at_index_by(pivot, |a, b| a.cmp(b));
+                    let (left, pivot, right) = v.partition_at_index_by(pivot, |a, b| a.cmp(b));
```

Comment:
Could be a good place to test that the lengths of left, pivot, right add up to the total length.

Reply:
Done.